src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/SwipeControlTests/SwipeControlTests.cs

```diff
@@ -31,15 +31,12 @@ public async Task When_SingleItem()
 
 			App.DragCoordinates(sutRect.CenterX, sutRect.CenterY, sutRect.Right - 10, sutRect.CenterY - 20);
 
-			await App.WaitForDependencyPropertyValueAsync(output, "Text", "Left_1");
+			await App.WaitForDependencyPropertyValueAsync(output, "Text", "** none **");
 		}
 
 		[Test]
 		[AutoRetry]
-		[ActivePlatforms(Platform.iOS, Platform.Android)]
-#if __SKIA__
-		[Ignore("Invalid layout of items")]
-#endif
```

Comment:
It's failing on Skia (the test is linked to runtime tests through https://github.com/unoplatform/uno/blob/master/src/Uno.UI.RuntimeTests/UITestsImport.props#L6).